pkg/expapi/v1/types.go

```diff
@@ -71,7 +71,9 @@ type SubresourceReference struct {
 
 // ResourceConsumption is an object for specifying average resource consumption of a particular resource.
 type ResourceConsumption struct {
-	Resource v1.ResourceName   `json:"resource,omitempty"`
+	// Resource specifies the name of the target resource.
+	Resource v1.ResourceName `json:"resource,omitempty"`
+	// Quantity specifies the target average consumption of the resource.
```

Comment:
This field may specifies also observed consumption. (It is present in spec and status).